src/Frontend/Modules/FormBuilder/EventListener/FormBuilderSubmittedMailSubscriber.php

```diff
@@ -35,6 +35,7 @@ public function onFormSubmitted(FormBuilderSubmittedEvent $event)
         if ($form['method'] == 'database_email') {
             // build our message
             $from = FrontendModel::getModuleSetting('Core', 'mailer_from');
+            $fieldData = $this->getEmailFields($event->getData());
```

Comment:
Happens in the ```getEmailFields()```method: https://github.com/jonasdekeukelaere/forkcms/blob/feature-formbuilder-cc/src/Frontend/Modules/FormBuilder/EventListener/FormBuilderSubmittedMailSubscriber.php#L86-L101